src/dnx.coreclr/tpa.cpp

```diff
@@ -49,10 +49,10 @@ BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
     }
     else
     {
-        pArray[0] = _wcsdup(L"Internal.Runtime.Uri.dll");
-        pArray[1] = _wcsdup(L"Internal.Uri.dll");
-        pArray[2] = _wcsdup(L"kre.coreclr.managed.dll");
-        pArray[3] = _wcsdup(L"kre.host.dll");
```

Comment:
Just to be sure the new copy of this file is still automatically generated, right?